### ISSUE TITLE
Orphanet X-refs on Mondo ID but none are "Mondo:equivalentTo"

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#4568]
**Mondo term (ID and Label):**
This affects multiple entries in Mondo, but a case example:
MONDO:0014389  
polyglucosan body myopathy 1 with or without immunodeficiency

**Xref that should be fixed (ID and label):**
Orphanet 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other comments:**
The data, as currently displaying in EBI, does not say that any of the X-refs on this term are equivalent mappings, per Mondo. So MedGen automated processing removed all OrphaIDs from our term with the Mondo ID since Mondo does not assert an exact match and we do not want to incorrectly display OrphaIDs on multiple terms in MedGen (when we get submissions to ClinVar or GTR based on external IDs like OMIM, Mondo or OrphaNet we have to represent those IDs once and only once in our data to support the Variant and Testing data from our users). But now it's unclear where we SHOULD map Orphanet IDs when Mondo doesn't assert an exact match in the data.

For this particular case example, all of the X-refs in Mondo appear to be referencing each other, but none seem to be equivalent to Mondo's term, and if I were just following these assertions, I would be stuck with zero external references/terms as equivalent to the Mondo concept. The ExactMatch links do help clarify during manual review, but for automated processing of Mondo data in MedGen, it would be great if the ExactMatch and X-refs were in agreement. (Disclaimer: I am not super familiar with our actual data processing of Mondo releases from the technical side, so if there is something other than the X-refs and equivalence statements therein that would aid in Mondo:Orpha matching as exact match based on IDs, I would be happy to pass it along to our dev team who may be able to modify our data processing. I am pretty sure we just updated it to account for the more thorough reporting of OrphaNet and OMIM relationships on parent concepts as well as the narrower children terms, I just don't know the details of that change)

**Comment by maglott** [comment]
To provide more technical background, MedGen just started to use only the skos:exactMatch values. We failed to determine how many XRefs would then go missing.

**Comment by matentzn** [comment]
@kanems @maglott Thank you for sharing this very important issue. Would it be possible if you or someone from your team could come to our technical call (Friday 9am PT)? 

- skos:exactMatch is only asserted if MONDO:equivalentTo (or obsoleteEquivalent) is asserted on the xref.
- I think you should not use xrefs at all, just skos:exactMatch
- I think Mondo should not have _any_ xrefs without a Mondo qualifier like equivalent, subclass or superclass (see below)

@nicolevasilevsky I think we should:

- add a QC check that forbids adding a mapping without a Mondo qualifier, at least for OMIM, ORDO, DO and NCIT.
- curate all the missing ones asap.

**Comment by maglott** [comment]
I can attend; this review exposed some gaps in our data flow as well so we cannot provide an accurate report right now of the XRefs we may have 'lost'.

**Comment by matentzn** [comment]
That would be great! I @nicolevasilevsky will send you the invite when she has time later today! 

Don't worry about details, this is mostly for us to touch base a bit and record mutual expectations and ideas :)

**Comment by kanems** [comment]
@nicolevasilevsky Can you include me on the invitation? I will also plan to attend, along with our team lead for MedGen, Adriana Malheiro (Donna or I can forward her the invite)

**Comment by nicolevasilevsky** [comment]
@maglott and @kanems I just invited you both to the call tomorrow, thanks! :)

**Comment by matentzn** [comment]
@kanems 

>  EBI, does not say that any of the X-refs on this term are equivalent mappings, per Mondo

You mean [here](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0014389):

<img width="382" alt="image" src="https://user-images.githubusercontent.com/7070631/156640637-4b74d3ea-6c68-4afd-a446-7505c27aebe5.png">

All xrefs in Mondo have annotations to say whether they are exact:


```
[Term]
id: MONDO:0014389
name: polyglucosan body myopathy 1 with or without immunodeficiency
xref: ICD10:E74.0 {source="MONDO:subClassOf", source="ORDO:397937/attributed", source="ORDO:397937/ntbt", source="Orphanet:397937"}
xref: OMIM:615895 {source="MONDO:subClassOf", source="MONDO:equivalentTo", source="ORDO:397937/ntbt", source="Orphanet:397937"}
xref: Orphanet:329173 {source="OMIM:615895", source="MONDO:relatedTo", source="MONDO:superClassOf"}
xref: Orphanet:397937 {source="OMIM:615895", source="MONDO:equivalentTo", source="MONDO:superClassOf"}
xref: UMLS:C4014605 {source="NCBI:mim2gene_medline", source="MONDO:equivalentTo"}
```

So what I don't understand: 

> So MedGen automated processing removed all OrphaIDs from our term with the Mondo ID since Mondo does not assert an exact match and we do not want to incorrectly display OrphaIDs on multiple terms in MedGen 

How did MedGen automated processing make that decision? Based on how OLS displays the mappings? There must me some mistake in my understanding of what you are experiencing!

In any case, your ticket led to an entirely different investigation, so it was very helpful, but your concrete issue, I do not understand.

**Comment by kanems** [comment]
@matentzn Yes, that screen shot of "Database cross references" in the EBI display is where I, as a curator, look first in Mondo to check equivalence connections in Mondo relative to other databases. How our workflow is specifically mapping the superClass, equivalent, btnt/ntbt, & other annotations you showed is more technical than I deal with as a part of my work so I'll get back to you on that. I can reiterate @maglott 's comment from above while I look for more details from our developers about the automated processes 

> MedGen just started to use only the skos:exactMatch values. We failed to determine how many XRefs would then go missing.

**Comment by matentzn** [comment]
Ok, thank you. in that case, it would be good to get a list of the xrefs your developers noted as lost. As for OLS, this is not really something we can deal with (as this is a display issue - the information is present), but as a work around, look (as a curator) only at the skos:exact match a bit lower down on the OLS side. These show the correct information. Looking forward to talking tomorrow!

**Comment by kanems** [comment]
Thanks for letting all of us jump onto your call today! I think we have sorted out the hiccups for the MedGen internal processing of ExactMatch and X-refs from Mondo, and it sounds like you all got a better picture of our processes and mappings.  So probably this specific ticket can be closed. 
We'll follow up soon with data regarding CUI:MondoID mapping and MIM#:MondoID mappings that we have stored which are discordant with Mondo's data.
